modules/installation-mirror-repository.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc
+
+[id="installation-mirror-repository_{context}"]
+= Mirroring the {product-title} image repository
+
+Mirror the {product-title} image repository to use during cluster installation
+or upgrade.
+
+.Prerequisites
+
+* You configured a mirror registry to use in your restricted network and
+can access the certificate and credentials that you configured.
+* You downloaded the pull secret from the
+link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
+page and modified it to include authentication to your mirror repository.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Review the
+link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Quay.io]
+repository to determine the version of {product-title} that you want to install.
+
+. Set the required environment variables:
++
+----
+$ export OCP_RELEASE=<release_version> <1>
+$ export LOCAL_REGISTRY='<bastion_host_name>:<local_registry_host_port>' <2>
```

Comment:
You don't have to mirror to a bastion; you could be mirroring to your corporate proxy or whatever for auditing/performance.  I think we should use `<local_registry_host_name>` or some such instead of `<bastion_host_name>`.  I guess it depends on how tightly we want to couple `modules/installation-mirror-repository.adoc` to `modules/installation-creating-mirror-registry.adoc` vs. allowing for pre-existing mirror registries.

Reply:
I'm adding some conditionals to limit "bastion" to the restricted network topic and generally refer to it as the "registry host" instead.